src/nfa/delta.cc

```diff
@@ -547,12 +547,12 @@ OrdVector<Symbol> Delta::get_used_symbols() const {
     //WITH BOOL VECTOR (error !!!!!!!):
     //return utils::OrdVector<Symbol>(utils::NumberPredicate<Symbol>(get_used_symbols_bv()));
 
-    //WITH BOOL VECTOR (1.9s):
-    std::vector<bool> bv{ get_used_symbols_bv() };
-    utils::OrdVector<Symbol> ov{};
-    const size_t bv_size{ bv.size() };
-    for (Symbol i{ 0 }; i < bv_size; ++i) { if (bv[i]) { ov.push_back(i); } }
-    return ov;
+    //WITH BOOL VECTOR (1.9s): (The fastest, it seems.)
```

Comment:
I would add some note for the reason it fails (epsilon is too large a number).

Reply:
Good idea. I added a comment.